testsuite/run_sets/build_validation_prepare_clm_filters.yml

```diff
@@ -0,0 +1,7 @@
+# This file describes the order of features in a Build Validation test suite run.
+
+## Add clm for rhel BEGIN ###
+
+- features/build_validation/add_custom_repositories/srv_prepare_clm_filters.feature
```

Comment:
Also if we mention that this is only for RH like, maybe we should rename the file acording.

Reply:
It's on server side, as the `srv_` prefix states it.

Besides, we don't have a prefix for all RH-like. The closest would be `allcli`.